src/jolt/subtable/test.rs

```diff
@@ -18,4 +18,22 @@ macro_rules! subtable_materialize_mle_parity_test {
       }
     }
   };
+  ($test_name:ident, $subtable_type:ty, $F:ty, $M:expr, $A:expr) => {
```

Comment:
Actually I think we can use a variadic here to avoid the duplication
```suggestion
  ($test_name:ident, $subtable_type:ty, $F:ty, $M:expr, $($constructor_args:expr),*) => {
```